Work around redefinition of sigcontext in libc headers

Introduced with:
https://github.com/torvalds/linux/commit/43d4da2c45b2f5d62f8a79ff7c6f95089bb24656

diff --git a/arch/arm64/include/uapi/asm/ptrace.h b/arch/arm64/include/uapi/asm/ptrace.h
index 98c4ce5..c44efed 100644
--- arch/arm64/include/uapi/asm/ptrace.h
+++ arch/arm64/include/uapi/asm/ptrace.h
@@ -23,8 +23,6 @@
 #include <linux/types.h>
 
 #include <asm/hwcap.h>
-#include <asm/sigcontext.h>
-
 
 /*
  * PSR bits
@@ -64,8 +62,6 @@
 
 #ifndef __ASSEMBLY__
 
-#include <linux/prctl.h>
-
 /*
  * User structures for general purpose, floating point and debug registers.
  */

